traceevents/src/test/java/com/tomtom/kotlin/traceevents/TracerTest.kt

```diff
@@ -361,6 +347,71 @@ class TracerTest {
         }
     }
 
+    class ClassWithoutToString {
+        @Suppress("unused")
+        val x = 1
+
+        @Suppress("unused")
+        val y = 2
+    }
+
+    class AnotherClassWithoutToString {
+        val a = 100
+    }
+
+    interface ListenerWithoutToString : TraceEventListener {
+        fun eventWithoutToString(classWithoutToString: Any)
+    }
+
+    class ConsumerWithoutToString : ListenerWithoutToString, TraceEventConsumer {
+        override fun eventWithoutToString(classWithoutToString: Any) {}
+    }
+
+    @Test
+    fun `register toString`() {
+        // GIVEN
+        val sutWithout = Tracer.Factory.create<TracerTest.ListenerWithoutToString>(this::class)
+        val consumerWithout = spyk(ConsumerWithoutToString())
```

Comment:
Was it deliberate that other variable names end in "WithoutToString" but this one is just "Without"?

Reply:
Yes.